docs/content/overview/product-telemetry.md

```diff
@@ -34,7 +34,9 @@ These are the data points collected and reported by NGINX Ingress Controller:
 - **TransportServers** The number of TransportServer resources managed by NGINX Ingress Controller.
 - **Replicas** Number of Deployment replicas, or Daemonset instances.
 - **Secrets** Number of Secret resources managed by NGINX Ingress Controller.
-- **Services** Number of Services referenced by VirtualServers, VirtualServerRoutes, TransportServers and Ingresses.
+- **ClusterIPServices** Number of ClusterIP Services managed by NGINX Ingress Controller.
+- **NodePortServices** Number of NodePort Services managed by NGINX Ingress Controller.
+- **ExternalNameServices** Number of ExternalName Services managed by NGINX Ingress Controller.
```

Comment:
@AlexFenlon we should add service type [LoadBalancer](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) as well

Reply:
@j1m-ryan actually asked me to remove this. I had LoadBalancer here initially.

Reply:
is that because its cloud provider specific not core to k8s?

Reply:
i misunderstood the pr, you can add it back